content/glossary/wikiops.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "WikiOps"
+description: 'Wiki documentation driven operation processes.'
+terms:
+- WikiOps
+- Wiki Operations
+tags:
+- wiki
+- docs
+- documentation
+- bad practice
+---
+WikiOps is the pattern of following Wiki documentation as the means of building and operating your infrastructure. This is to be avoided as it relies on ClickOps and documentation that becomes quickly outdated and inconsistent.
```

Comment:
```suggestion
WikiOps is the pattern of following Wiki-style documentation as the means of building and operating your infrastructure. This is to be avoided as it relies on ClickOps and documentation that becomes quickly outdated and inconsistent.
```

Reply:
Have we defined `ClickOps`? that's another good one.

Reply:
We have not. I will add 👍